tempto-core/src/main/java/io/prestosql/tempto/internal/fulfillment/table/hive/HiveTableManager.java

```diff
@@ -200,7 +203,15 @@ private String getImmutableTableHdfsPath(HiveDataSource dataSource)
 
     private String getMutableTableHdfsPath(TableName tableName, Optional<Integer> partitionId)
     {
-        String location = hiveThriftClient.getLocation(tableName);
+        QueryResult queryResult = queryExecutor.executeQuery("SHOW CREATE TABLE " + tableName.getNameInDatabase());
+        String value = (String) queryResult.row(0).get(0);
+        Pattern locationPattern = Pattern.compile(".*LOCATION\\W+'([^']+)'.*");
+        Matcher matcher = locationPattern.matcher(value);
+        if (!matcher.matches()) {
```

Comment:
remove `.*` from the pattern, and use `match.find()` instead of `matcher.matches()`

(also, `.*` is not sufficient unless you enable DOT_ALL)